tests/framework/microvm.py

```diff
@@ -993,6 +993,11 @@ def open(self):
             # serial already opened
             return
 
+        attempt = 0
+        while not Path(self._vm.screen_log).exists() and attempt < 5:
+            time.sleep(0.2)
+            attempt += 1
+
```

Comment:
Should we continue knowing there is no log still? or do a final `assert self._vm.screen_log.exists()`?

Reply:
I can't really see a difference, to be honest. It will fail either way. I used a pattern from https://github.com/firecracker-microvm/firecracker/blob/main/tests/framework/utils_vsock.py#L150 .